imports/client/components/App.tsx

```diff
@@ -65,21 +70,21 @@ const BreadcrumbList = styled.ol`
 `;
 /* stylelint-enable value-no-vendor-prefix */
 
-const BreadcrumbItem = styled.li`
+const BreadcrumbItem = styled.li<{ theme: Theme }>`
```

Comment:
If this parameterization is required for type checking, we should probably have it everywhere `theme` is used.  If not, we should probably have it nowhere?